libr/cons/cons.c

```diff
@@ -469,6 +469,10 @@ R_API RCons *r_cons_new() {
 	I.null = 0;
 #if __WINDOWS__
 	I.ansicon = r_sys_getenv ("ANSICON");
+#if UNICODE
+	SetConsoleOutputCP (CP_UTF8);
```

Comment:
Same here, set only if UTF8 is listed in https://docs.microsoft.com/en-us/windows/desktop/api/winnls/nf-winnls-enumsystemcodepagesa

Reply:
Also please check for the error, since it might fail:

> If the function succeeds, the return value is nonzero.
> If the function fails, the return value is zero. To get extended error information, call `GetLastError()`.